app/src/pages/Home.tsx

```diff
@@ -20,12 +21,13 @@ import { breakpoints, pxToRem } from '../styled-theme';
 
 export const Home: React.FC = () => {
   const dispatch = useAppDispatch();
+  const peersTableOptions = useAppSelector(getPeersTableOptions);
```

Comment:
Can you confirm that it's still showing the correct number of total peers on the validators/peers info card on this page?